### PR TITLE
Flag new stream

### PR DESCRIPTION
**Pull request by ninehundred**
pull request to add flag to cards on landing (dashboard) if the most recent main is ahead of the most recent actcarbon branch.

### CODE REVIEW

File: src/components/landing/ProjectCard.vue [hunk 1]
```diff
@@ -26,26 +27,45 @@
...
+        </v-card-actions>
+        <v-overlay class="d-inline-flex pa-2" :absolute="true" :opacity="0.98" :value="cardOverlay">
+          <v-col class="d-flex flex-column align-end mb-6">
+            <v-btn icon color="red darken-1" @click="cardOverlay = false">
+              <v-icon>mdi-close-circle</v-icon>
+            </v-btn>
+          </v-col>
+          <v-col>
+            <v-card-text>
+              Theres an update on main branch, use edit to update your report.
```
Comment (tomhebunn) [inline src/components/landing/ProjectCard.vue:65]: needs rewording if edit is not yet available

**Review by Exilliar — COMMENTED** [review]
Looking good! Just a couple of comments and I think that it would probably be good if the "x" button on the flag was at the top right of the card instead of just above the text

File: src/views/Landing.vue [hunk 2]
```diff
@@ -264,7 +301,9 @@ export default class Landing extends Vue {
           co2Values: co2Data,
           totalCO2e: proj.data.data.stream.object.data.totalCO2,
           link: "",
-          category: proj.data.data.stream.object.data.projectData.components,
+          category: `${proj.data.data.stream.object.data.projectData.components}`,
```
[the remark this answers is not in the document]
Comment (ninehundred) [reply src/views/Landing.vue:304]: the Project ts datatype notes this as being a string.

[the remark this answers is not in the document]
Comment (Exilliar) [reply src/views/Landing.vue]: I think that if you do a pull from dev then the interface should get updated to be an array of strings (was done in Robert's latest pr)

[the remark this answers is not in the document]
Comment (ninehundred) [reply src/views/Landing.vue]: ok! pulled from dev and incorporated into the branch. thanks for the pointers!

**Review by Exilliar — COMMENTED** [review]
Looking good! I think there's just one comment from @tomhebunn that needs resolving and it would also be nice if the "x" was in the very top right of the card (hopefully shouldn't be too difficult). I'm also getting some error messages whenever I click on the alert icon which look like they're to do with the chart, but that's something that can be fixed in another issue

**Comment by ninehundred** [comment]
moved the closer to top right

**Review by Exilliar — APPROVED** [review]

**Pull request merged**